alembic/versions/22fb4e1e3139_add_ticket_client.py

```diff
@@ -0,0 +1,37 @@
+"""add ticket client
+
+Revision ID: 22fb4e1e3139
+Revises: 40c4ad7c0909
+Create Date: 2015-05-13 16:51:14.399678
+
+"""
+
+revision = '22fb4e1e3139'
+down_revision = '40c4ad7c0909'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_table('ticket_client',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('created_at', sa.DateTime(), nullable=False),
+        sa.Column('updated_at', sa.DateTime(), nullable=False),
+        sa.Column('name', sa.Unicode(length=80), nullable=False),
+        sa.Column('client_event_id', sa.Unicode(length=80), nullable=False),
+        sa.Column('client_id', sa.Unicode(length=80), nullable=False),
```

Comment:
Code convention: we use the `_id` suffix to indicate a foreign key reference. Since these refer to external ids with no database constraint, they should be either (a) without the `_` (so just `clientid`, similar to `userid` for the buid key in the User model) or using some other suffix (like `_key`).